docpages/advanced_reference/coroutines.md

```diff
@@ -1,225 +1,227 @@
-                    co_return member->second;
-
-                // Try looking in guild cache
-                dpp::guild *guild = dpp::find_guild(event.command.guild_id);
-                if (guild) {
-                    // Look in guild's member cache
-                    if (auto member = guild->members.find(user_id); member != guild->members.end()) {
-                        co_return member->second;
-                    }
-                }
-
-                // Finally if everything else failed, request API
-                dpp::confirmation_callback_t confirmation = co_await event.from->creator->co_guild_get_member(event.command.guild_id, user_id);
-                if (confirmation.is_error())
-                    co_return std::nullopt; // Member not found, return empty
-                else
-                    co_return confirmation.get<dpp::guild_member>();
-            };
-
-            // Send a "<bot> is thinking..." message, to wait on later so we can edit
-            dpp::async thinking = event.co_thinking(false);
-
-            // Call our coroutine defined above to retrieve the member requested
-            std::optional<dpp::guild_member> member = co_await resolve_member(event);
-
-            if (!member.has_value()) {
-                // Wait for the thinking response to arrive to make sure we can edit
-                co_await thinking;
-                event.edit_original_response(dpp::message{"User not found in this server!"});
-                co_return;
-            }
-
-            std::string avatar_url = member->get_avatar_url(512);
-            if (avatar_url.empty()) { // Member does not have a custom avatar for this server, get their user avatar
-                dpp::confirmation_callback_t confirmation = co_await event.from->creator->co_user_get_cached(member->user_id);
-
-                if (confirmation.is_error())
-                {
-                    // Wait for the thinking response to arrive to make sure we can edit
-                    co_await thinking;
-                    event.edit_original_response(dpp::message{"User not found!"});
-                    co_return;
-                }
-                avatar_url = confirmation.get<dpp::user_identified>().get_avatar_url(512);
-            }
-
-            // Wait for the thinking response to arrive to make sure we can edit
-            co_await thinking;
-            event.edit_original_response(dpp::message{avatar_url});
-        }
-    });
-
-
-    bot.on_ready([&bot](const dpp::ready_t & event) {
-        if (dpp::run_once<struct register_bot_commands>()) {
-            dpp::slashcommand command("avatar", "Get your or another user's avatar image", bot.me.id);
-
-            command.add_option(dpp::command_option(dpp::co_user, "user", "User to fetch the avatar from"));
-
-            bot.global_command_create(command);
-        }
-    });
-
-    bot.start(dpp::st_wait);
-}
-~~~~~~~~~~
+\page coroutines Advanced Commands with Coroutines
```

Comment:
I cannot review this file. I suspect the culprit is line endings? I wrote this on Windows but I expected git to normalize the line feeds to \n.

Regardless this will be undergoing huge changes from my recent PR, I suggest unstaging this for now

Reply:
Can always also check live docs vs the PR netlify. But yeah, given the major rewrite of coro docs incoming, I may hold on to that.